### PR TITLE
preserve viewbox option [fixes #8]

### LINKED ISSUE #8: preserve `viewbox` attribute
I see the template is generating a viewbox from `0 0 {width} {height}`

Some of the SVG source files I use don't have a width and height, but they do have a `viewBox`. They are being rendered as symbols with `viewBox="0 0"` — I'd rather they kept their viewBox.

Is that easy to do?

thanks :)

### PR DESCRIPTION
**Pull request by bboyle**
- new option uses existing `svg[viewBox]` attribute value for generated
  symbol
- updated tests
- added 2 new fixtures with conflicting width/height/viewBox values
- update internal template to use viewBox parameter
- update task script to generate viewBox from width and height when
  viewBox option not specified (or viewBox attribute not found)
- updated readme
- added grunt to devDependencies to fix local tests
- update version to v0.2.0 ([MINOR version when you add functionality in a backwards-compatible manner](http://semver.org/))

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -70,6 +70,14 @@ module.exports = function(grunt) {
         files: {
           'tmp/remove_attrs': ['test/fixtures/mail.svg', 'test/fixtures/lock.svg']
         }
+      },
+      preserve_viewbox: {
+        options: {
+          preserveViewbox: true
```
Comment (Lorti) [inline Gruntfile.js]: Can we camelCase this option to `preserveViewBox`? Otherwise your code seems great, looking forward to publish an update 👍

Comment (bboyle) [reply Gruntfile.js]: sure :)

**Comment by bboyle** [comment]
sweet! @Lorti can you publish this update on npm? :)

**Comment by Lorti** [comment]
I have cleaned the `package.json` file to match the core plugins and then published your update. 
Thanks! Also, I really like your Animation Mentor reel ;)

**Pull request merged**